src/main/java/dp/handler/Handler.java

```diff
@@ -49,60 +63,138 @@
 
     @KafkaListener(topics = "${KAFKA_TOPIC:common-output-created}")
     public void listen(final ExportedFile message) {
-
-        LOGGER.info("exporting file to xlsx using filterID: {}", message.getFilterId());
-
+        MessageType messageType = GetMessageType(message);
         final AmazonS3URI uri = new AmazonS3URI(message.getS3URL().toString());
 
-        Workbook workbook = null;
         try (final S3Object object = s3Client.getObject(uri.getBucket(), uri.getKey())) {
+            if (FILTER.equals(messageType)) {
+                handleFilterMessage(message, object);
+            } else {
+                handlePrePublishMessage(message, object);
+            }
+        } catch (final IOException | FilterAPIException e) {
+            if (FILTER.equals(messageType)) {
+                LOGGER.error("error when exporting filter {}. exception : {}", message.getFilterId(), e);
+            } else {
+                LOGGER.error("error when exporting pre publish message {}. exception : {}",
+                        message.getFilename().toString(), e);
+            }
+            return;
+        } catch (Exception e) {
+            LOGGER.error("unexpected error throw while attempting to process message, {}", e.getMessage());
```

Comment:
Just pass the exception to the logger, by calling `e.getMessage()` you wont get a stack trace